components/st-microelectronics/landpatterns.stanza

```diff
@@ -267,12 +267,12 @@ public pcb-landpattern LQFP80:
                       tol(0.6, 0.15),  ;Lead size in x direction:   L
                       tol(0.22, 0.05)) ;Lead size in y direction:   b
 
-;public pcb-landpattern UFQFPN20:
-;  make-qfn-landpattern(20,             ;Number of pins
-;                      0.5,             ;Pitch:                      e
-;                      tol(3.0, 0.1),   ;Outer size (lead-to-lead):  D and E
-;                      tol(0.55, 0.05), ;Lead size in x direction:   L1?
-;                      tol(0.25, 0.05)) ;Lead size in y direction:   b
+public pcb-landpattern UFQFPN20:
+  make-qfn-landpattern(20,             ;Number of pins
+                      0.5,             ;Pitch:                      e
+                      tol(3.0, 0.1),   ;Outer size (lead-to-lead):  D and E
+                      tol(0.55, 0.05), ;Lead size in x direction:   L1?
```

Comment:
L,  not L1. It's the size of the lead that makes contact with the board, not the space from the lead to the packaging.

Reply:
> L, not L1. It's the size of the lead that makes contact with the board, not the space from the lead to the packaging.

It is actually L1. However, removed this code since the given function will not create pins of correct length at the corners. Will add it later